pkgs/applications/science/math/geogebra/default.nix

```diff
@@ -57,7 +29,59 @@ stdenv.mkDerivation rec {
     homepage = "https://www.geogebra.org/";
     maintainers = with maintainers; [ ma27 ];
     license = with licenses; [ gpl3 cc-by-nc-sa-30 geogebra ];
-    platforms = platforms.all;
+    platforms = with platforms; linux ++ darwin;
```

Comment:
This doesn't evaluate since `darwin` isn't from `platforms.darwin`, but from the `darwin`-attribute in your expression since it's a recursive attr-set (due to the `rec` keyword).

You can check the validity of `meta`-sections by setting `checkMeta = true;` in your `~/.config/nixpkgs/config.nix` btw :)

Reply:
> You can check the validity of meta-sections by setting checkMeta = true; in your ~/.config/nixpkgs/config.nix btw :)

Thanks, didn't know about this option.